packages/sync/src/useMultiplayerSync.ts

```diff
@@ -155,13 +167,32 @@ export function useMultiplayerSync(
 	)
 }
 
-/** @public */
+/**
+ * Options for the {@link useMultiplayerSync} hook.
+ * @public
+ */
 export interface UseMultiplayerSyncOptions {
+	/**
+	 * The URI of the multiplayer server. This must include the protocol, e.g. `wss://server.example.com` or `http://localhost:5858`.
+	 */
 	uri: string
-	roomId?: string
+	/**
+	 * The user information. If not provided, a default implementation based on localStorage is used.
+	 * This should be synchronized with the configuration for the main `<Tldraw />` component.
+	 */
 	userPreferences?: Signal<TLUserPreferences>
-	/* @internal */
-	trackAnalyticsEvent?(name: string, data: { [key: string]: any }): void
+	/**
+	 * The asset store for blob storage. See {@link tldraw#TLAssetStore}.
+	 * If not provided assets will be stored as base64 strings in the document, which can cause performance issues at serialization boundaries.
+	 */
 	assets?: Partial<TLAssetStore>
```

Comment:
is it worth:
1. making this required & not partial
2. renaming `defaultAssetStore` to `inlineBase64AssetStore` or `slowBase64AssetStore` something
3. export it and say you can use it if you want for prototyping maybe

Reply:
Did this and in the process I made `resolve` optional, since it seems like that's an advanced-level feature. lemme know if I should revert.